homeassistant/components/volvo/strings.json

```diff
@@ -104,7 +104,12 @@
         "name": "Charging power"
       },
       "charging_power_status": {
-        "name": "Charging power status"
+        "name": "Charging power status",
+        "state": {
+          "providing_power": "Providing power",
+          "no_power_available": "No power",
+          "unknown": "Unknown"
```

Comment:
we should not return `Unknown` or provide it as a state, instead, make the sensor return `None` and omit it from the `options`